extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+        color1: "#772CE8",
+        menuIconURI: "https://tekinical.github.io/twitchURI.png",
+        docsURI:
+          "https://github.com/Tekinical/tekinical.github.io/blob/main/Twitch%20Extension%20Doc.md",
+        blocks: [
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Twitch Chat",
+          },
+          {
+            opcode: "setupchat",
+            text: "setup chat [username] chat fade enabled [fade] show bots [bots] prevent clipping [prevent] width [width] height [height] position [position]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              username: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "Twitch Username",
+              },
+              fade: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 30,
+              },
+              bots: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: true,
+              },
+              prevent: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              width: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "100%",
+              },
+              height: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "100%",
+              },
+              position: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "absolute",
+              },
+            },
+          },
+          {
+            opcode: "closechat",
+            text: "close chat",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Overlays",
+          },
+          {
+            opcode: "text",
+            text: "create text overlay with text [text] with italic [italic] with bold [bold] with size [size] with font [font]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              text: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "hi chat",
+              },
+              italic: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              bold: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              size: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "20px",
+              },
+              font: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "MS Comic Sans",
```

Comment:
This should be a menu/dropdown that only shows fonts that are loaded directly into TurboWarp.

Take a look at how Animated Text does this for reference.

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/b4ced62a-be99-44ea-b0d8-cbc155d0300d)
![image](https://github.com/TurboWarp/extensions/assets/140775902/6f59be6a-c63b-4ecb-a9e9-23e10ca435c3)
credited you too 
![image](https://github.com/TurboWarp/extensions/assets/140775902/a8761485-b7bc-44f5-a603-a71865a1a2da)